src/components/HomePageContent/DropZoneSection.tsx

```diff
@@ -65,6 +65,23 @@ const DropZoneSection = styled(({ className, updateCertificate }: DropZoneSectio
         });
     });
 
+    //if there is a parent window, send a READY to parent
+    if (window.opener) {
+      window.opener.postMessage({ type: "READY" });
+    }
+
+    //detecting message from parent window
+    window.addEventListener(
+      "message",
+      (event) => {
+        if (event.data.type == "LOAD_DOCUMENT") {
```

Comment:
```suggestion
        if (event.data?.type === "LOAD_DOCUMENT") {
```

This prevents crashing if there were message sent to this window by accident and crashes the code.